python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -1018,6 +1020,13 @@ def _node_resources(self, node_id):
         else:
             return {}
 
+    def _node_labels(self, node_id):
```

Comment:
Maybe node_labels_from_config? Can this return labels passed by the env var?

Reply:
No. This is from cluster.yaml. I name it this way to be consistent with `_node_resources`